cookie-banner-rules-list.json

```diff
@@ -893,6 +893,1279 @@
+          }
+        ]
+      },
+      "id": "1dc10ed2-800f-484c-9df1-6f99d8cd2584"
+    },
+    {
+      "click": {
+        "optIn": "button.js-accept",
+        "presence": "div.cookie-banner-buttons"
+      },
+      "domain": "emag.ro",
+      "cookies": {},
+      "id": "dbbccc0a-13ba-4cd8-9cf0-32420401be55"
+    },
+    {
+      "click": {
+        "optIn": "button.CookieConsent_button__FJrIc",
+        "presence": "div.CookieConsent_holder__ZsT3G"
+      },
+      "domain": "kupujemprodajem.com",
+      "cookies": {
+        "optIn": [
+          {
+            "name": "FCNEC",
+            "value": "%5B%5B%22AKsRol-hQ0GC8scayCxS70oIbLW375lxZBB419tjfwu1fOoWf-QLr5NOEOgKKLPu5Em82ydIwG4STH2DOpPVM65ZS_y2CCQlVsaUDCu-KZl2LrrYKqTftQDxBBuO-EmausJD4fO6ZHNWJEHTyKM8wJgrIadkxR4m9g%3D%3D%22%5D%2Cnull%2C%5B%5D%5D"
+          }
+        ],
+        "optOut": [
+          {
+            "name": "FCCDCF",
+            "value": "%5Bnull%2Cnull%2Cnull%2C%5B%22CPgX9oAPgX9oAEsABCENCjCgAAAAAAAAABpYAAAOhQD2F2K2kKEkfjSUWYAQBCujIEIBUAAAAECBIAAAAUgQAgFIIAgAAlACAAAAABAQAQCAgAQABAAAoACgACAAAAAAAAAAAAQQAABAAIAAAAAAAAEAQAAAAAQAAAAAAABEhCAAQQAEAAAAAAAAAAAAAAAAAAABAAA.YAAAAAAAAAA%22%2C%221~%22%2C%22D2A36A86-FAFB-4BB0-951A-2BA964C5C018%22%5D%2Cnull%2Cnull%2C%5B%5D%5D"
+          }
+        ]
+      },
+      "id": "944c0cf0-271b-41f8-b0ca-3fa67db7af29"
+    },
+    {
+      "click": {
```

Comment:
Please make sure we have click rules for all google rules, including our existing google records.

Reply:
Fixed all Google click rules.